integration-core/src/main/java/org/symphonyoss/integration/core/bootstrap/IntegrationBootstrapContext.java

```diff
@@ -335,6 +337,16 @@ private void setupIntegration(IntegrationBootstrapInfo info) {
     }
   }
 
+  private void checkRetryAttempt(IntegrationBootstrapInfo integrationInfo, String integrationUser, RetryLifecycleException e) {
```

Comment:
You could retrieve the integration username from the IntegrationBootstratpInfo object rather than pass it as a parameter.